privacyidea/app.py

```diff
@@ -196,37 +172,42 @@ def create_app(config_name="development",
 
     app.response_class = PiResponseClass
 
+    # Setup logging
+    have_logging_conf = False
     try:
         # Try to read logging config from file
         log_config_file = app.config.get("PI_LOGCONFIG",
                                          "/etc/privacyidea/logging.cfg")
         if os.path.isfile(log_config_file):
             logging.config.fileConfig(log_config_file)
+            have_logging_conf = True
             if not silent:
-                print("Reading Logging settings from {0!s}".format(log_config_file))
-        else:
-            raise Exception("The config file specified in PI_LOGCONFIG does "
-                            "not exist.")
+                print("Read Logging settings from {0!s}".format(log_config_file))
+        # now try to read from yaml config
+        log_conf_yaml = app.config.get("PI_LOGCONFIG_YAML",
```

Comment:
But this means, if I have both in my pi.cfg
* PI_LOGCONFIG
* PI_LOGCONFIG_YAML
both the cfg file and the yaml files are read. Does this make sense?
Also, If I first successfully read the cfg file but do an exception during yaml read, I would have a valid cfg file config, but still enter the ``except``.  OK. But sind have_logging_conf is true, I would actually use the cfg file.

At first glance this looks a bit weired to me, since several things are tried and if something works out, it is used.
I would rather expect that I **define** to use yaml config and it fails I would not want a fallback on the cfg file.
Hm, in this case I would simply have to remove ``PI_LOGCONFIG`` from the pi.cfg. OK, technically fine.

But this is a strange behaviour, which needs a good documentation.

Reply:
Yeah, i was trying to be backwards compatible as much as possible but i think we should just use an either/or approach:
- Have only one PI_LOGCONFIG in the config file
- detect whether it is a conf or yaml file and use the appropriate reader
I'll update this.

Reply:
detecting the file type would be totally awesome.
Maybe we can try to load the file as a yaml file (new behaviour) and if it thows an exception load it as cfg (old behaviour)

Reply:
I've added a detection based on the ending.

Reply:
I am not that happy with it. See my  new comments.